src/amo/components/HomeHeroBanner/index.js

```diff
@@ -147,7 +147,64 @@ export class HomeHeroBannerBase extends React.Component<Props> {
 
           <p>
             {i18n.gettext(`Simple zoom in/out tool for a close-up view of any
-              web content.`)}
+              web content`)}
+          </p>
+        </HeroSection>
+      ),
+      (
+        <HeroSection
+          key="hero-12"
+          linkTo="/addon/wikiwand-wikipedia-modernized/"
+        >
+          <h3>{i18n.gettext('Wikiwand')}</h3>
+
+          <p>{i18n.gettext('Give Wikipedia a gorgeous makeover')}</p>
+        </HeroSection>
+      ),
+      (
+        <HeroSection
+          key="hero-13"
+          linkTo="/addon/onetab/"
+        >
+          <h3>{i18n.gettext('OneTab')}</h3>
+
+          <p>
+            {i18n.gettext(`Convert your open tabs into a list and save
+              precious memory space`)}
```

Comment:
Editorial comment here (cc @devaneymoz): "precious memory space" is to me strange and somewhat confusing phrasing. Maybe just "precious memory"? Or "... into a list and reduce memory usage"?

Reply:
@tofumatt sure, I like your suggested edit. Let's kill the "space"; I'll update content doc, as well.